webapp/pages/index.vue

```diff
@@ -92,9 +92,39 @@ export default {
       ],
     }
   },
+
+  mounted() {
+    if (this.hashtag) {
+      this.changeFilterBubble({ tags_some: { name: this.hashtag } })
+    }
+    if (!this.hasAgreedToLatestTermsAndConditions) {
+      this.$router.push('terms-and-conditions')
+    }
+  },
+
   computed: {
     ...mapGetters({
+      currentUser: 'auth/user',
+      posts: 'posts/posts',
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:49:53Z" title="Friday, August 16th 2019, 10:49:53 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -92,9 +92,39 @@ export default {
       ],
     }
   },
+
+  mounted() {
+    if (this.hashtag) {
+      this.changeFilterBubble({ tags_some: { name: this.hashtag } })
+    }
+    if (!this.hasAgreedToLatestTermsAndConditions) {
+      this.$router.push('terms-and-conditions')
+    }
+  },
+
   computed: {
     ...mapGetters({
+      currentUser: 'auth/user',
+      posts: 'posts/posts',
```


```suggestion
```
I think this does not exist anymore